### ISSUE TITLE
Indicate that Solid can be used by organizations

### ISSUE THREAD

**Issue by lecoqlibre** [solid/solidproject.org#813]
When browsing the website, some people might think that Solid is designed to manage personal data only like the term "POD" suggests it.

I would like to propose some changes to the text that clearly say that Solid can be used to manage both personal and organizations data.

This will show that Solid can be used as a "data space", which is a strongly discussed concept in the EU.

If you think it's a good idea, I be pleased to propose a PR.

**Comment by csarven** [comment]
Makes sense. Though I'd suggest more along the lines of "individuals and communities".

**Comment by lecoqlibre** [comment]
Thanks for your feedback, I just opened a PR using "individuals and communities" like you suggested.

**Comment by TallTed** [comment]
I'm afraid that "individuals and communities" suggests it's not usable by "businesses". "Groups" is more general, more inclusive.

So, "people" might be better made "individuals and groups, including businesses" or even "individuals and groups, large and small, including businesses".

**Comment by lecoqlibre** [comment]
Good point @TallTed, one should definitely understand that Solid is also usable by businesses. I think it's important for politicians. For instance, Gaia-X was presented in 2019 by the German and French Ministries of Economic Affairs.

@csarven Why did you propose "communities" instead "organizations"?

An alternative would be to use the three like "individuals, communities and businesses"?

**Comment by csarven** [comment]
I said communities because I find it more fluid and not as constrained or necessarily formal as organisations or groups. Solid always had some emphasis on "social", so communities is a way to reflect that. Groups works too. I find businesses overly specific... especially for the slot of 2-3 words. Organisations would be preferable to businesses, and covers it.

Sure the following terms (and many more) are relevant for use:

* individuals
* communities
* people
* groups
* organisations
* businesses
* institutions
* governments
* entities
* collectives

**Comment by lecoqlibre** [comment]
OK so maybe we could use "individuals and organisations" to stay concise on the home page and detail how Solid can be used for communities, businesses, governments, etc on the "About" page?

The "About" page should be edited anyway to replace "people" by "individuals and ..." like on the home page to stay consistent.

**Comment by csarven** [comment]
I should've added: if it is between organisations and groups, prefer groups. Group generally covers organisation. Lastly, the subject is about the collection of individuals, as opposed to an organisation as an entity. Again, a touch attempting to retain the social aspect of Solid FWIW.

**Comment by lecoqlibre** [comment]
Thanks @csarven I've changed organisations to groups and re-requested a review from @TallTed.

**Comment by csarven** [comment]
Thanks. Is this issue now resolved by merging of PR #814 ?

**Comment by lecoqlibre** [comment]
@csarven In an [earlier comment](https://github.com/solid/solidproject.org/issues/813#issuecomment-1809158103) of this issue, it seems that we should have some use cases showed in the "about" page like "how Solid can be used for communities, businesses, governments, etc". Should I open a new issue or keep that one open?

**Comment by csarven** [comment]
I don't have an opinion on whether that kind of information belongs in /about or somewhere else at the moment but makes sense to track it separately. Perhaps needs to incorporate/factor-in some stuff from https://github.com/solid/user-stories .

**Comment by michielbdejong** [comment]
@lecoqlibre do you feel this is improved by [the redesign](https://github.com/solid/solidproject.org/pull/891)? If not, can you indicate how you would edit the html content in the `2023-redesign` branch of this repo to improve it? Preferably in a PR! :)
